fdmi/ut/sd_send_not.c

```diff
@@ -244,9 +245,21 @@ void fdmi_sd_send_notif(void)
 	struct m0_rpc_conn_pool      *conn_pool;
 	struct m0_rpc_conn_pool_item *pool_item;
 	int                           rc;
+	static int                    tested_number = 0;
 
-	fdmi_serv_start_ut(&filterc_send_notif_ops);
+	if (tested_number++ > 0) {
+		/* FIXME: This UT can not run repeatedly. */
+		return;
+	}
```

Comment:
I am still trying to fix this.
This is a workaround. So, I placed a FIXME here.

Reply:
This is fixed in recent commit.